src/aiida/manage/configuration/config.py

```diff
@@ -528,6 +528,7 @@ def create_profile(
 
         LOGGER.report('Initialising the storage backend.')
         try:
+            # PRCOMMENT: Not sure what this context manager is for?
```

Comment:
This context manager change sys.stdout here temporary, so this it is doing `sys.stdout = mystdout`. Since it does not store the redirected information anywhere, I think person just wanted to prevent whatever `profile.storage_cls.initialise(profile)` is outputting is printed to the user terminal.

Reply:
ah you already have remove it in a PR. Okay i see your question was more about why do we redirect this and not print it to the terminal

Reply:
I am a bit afraid that the removal now triggers stdout at places where you actually don't want it to happen, because we use this function at places where the user would be confused about migration. Check usage of `create_profile`. For example now `verdi presto` prints information about initialisation. I think i in verdi presto the additional prints are fine and the other usages I seem also fine but I did not thoroughly checked. Hope you checked it when merging the PR.